benchmark/src/main/java/com/amazon/opendistroforelasticsearch/sql/benchmark/BenchmarkService.java

```diff
@@ -67,28 +73,30 @@ private BenchmarkService(final String filePath) throws Exception {
 
   /**
    * Function to run all procedures related to benchmarking.
+   *
    * @throws Exception Thrown if benchmarking fails.
    */
   private void runBenchmarks() throws Exception {
     final List<BenchmarkResults> results = new ArrayList<>();
-    for (Double sf: scaleFactors) {
+    for (Double sf : scaleFactors) {
       DataGenerator.generateData(benchmarkPath, sf);
-      QueryGenerator.generateQueries(benchmarkPath, sf);
-      for (final String type: types) {
+      // QueryGenerator.generateQueries(benchmarkPath, sf);
+      for (final String type : types) {
         DatabaseLauncher launcher = DatabaseLauncherFactory.getDatabaseLauncher(type);
```

Comment:
If you make DatabaseLauncher AutoCloseable (https://docs.oracle.com/javase/8/docs/api/java/lang/AutoCloseable.html), then you can use a try-with-resources block to auto-close it.

Reply:
To close/stop the database,  systemctl stop command needs to be executed.

Reply:
You can still do that using an AutoCloseable.

Reply:
Thanks. Added this change.